packages/cli/src/components/ProjectStatus/ClientProjectStatus.tsx

```diff
@@ -58,7 +58,7 @@ export default function ClientProjectStatus({
         break
 
       case "BUILD_FAILED":
-        setEvent(["error", "Could not build client"])
+        setEvent(["error", "Could not build client" + event.errorMessage])
```

Comment:
```suggestion
        setEvent(["error", "Could not build client: " + event.errorMessage])
```

Reply:
Oops, it was not my intention to commit this... reverted it.